x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/min_age_field/min_age_field.tsx

```diff
@@ -81,9 +81,43 @@ interface Props {
 }
 
 export const MinAgeField: FunctionComponent<Props> = ({ phase }): React.ReactElement => {
+  const minAgeValuePath = `phases.${phase}.min_age`;
+  const minAgeUnitPath = `_meta.${phase}.minAgeUnit`;
+
   const { isUsingRollover } = useConfiguration();
+  const globalFields = useGlobalFields();
+
+  const { setValue: setMillisecondValue } = globalFields[
+    `${phase}MinAgeMilliSeconds` as 'coldMinAgeMilliSeconds'
```

Comment:
I'm not sure why we cast the field path as `coldMinAgeMilliSeconds` here, it can be other phases too, right?

Reply:
They all cast to `number` so it is not necessary to write `as 'warmMinAgeMilliSeconds' | 'coldMinAgeMilliSeconds' | 'frozenMinAgeMilliSeconds' | 'deleteMinAgeMilliSeconds'`